fastlane/Gemfile

```diff
@@ -1,24 +1,3 @@
-source 'https://rubygems.org'
-
-# Specify your gem's dependencies in .gemspec
-gemspec
-
-if `cd ..; git remote -v`.include?("countdown")
```

Comment:
@mfurtak we kind of already had this, in a quite hacky way before, but only for the _fastlane_ tool